### PR TITLE
Add an explain section to update symfony recipes

### PR DESCRIPTION
**Pull request by maxhelias**
More documentation about new command for update recipe

### CODE REVIEW

**Comment by maxhelias** [comment]
Finally, I just copy/past the section of the minor guide 😅 
https://symfony.com/doc/current/setup/upgrade_minor.html#updating-recipes

**Comment by maxhelias** [comment]
friendly ping @weaverryan 😄

**Comment by weaverryan** [comment]
Sorry I missed this. It makes total sense. When we have duplicated sections like this, we can isolate that section into its own file and use `include::` to include it - see `reference/constraints/Choice.rst` for an option it includes `/reference/constraints/_payload-option.rst.inc` (we always end the included files in .inc so that they are not parsed as standalone files). Could you isolate this section into a .inc file and include from both documents?

Thanks!

**Comment by maxhelias** [comment]
poke @weaverryan 😉

File: setup/_update_recipes.rst.inc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+3) Updating Recipes
+-------------------
+
+Over time - and especially when you upgrade to a new version of a library - an
+updated version of the :ref:`recipe <recipes-description>` may be available.
+These updates are usually minor - e.g. new comments in a configuration file - but
+it's a good idea to update the core Symfony recipes.
```
Comment (wouterj) [inline setup/_update_recipes.rst.inc:7]: Actually, contrib recipes as well. What about a minor reword: "it's a good idea to keep your files in sync with the recipes" (or something the like)

**Review by wouterj — APPROVED** [review]
Looks good!

File: setup/_update_recipes.rst.inc (hunk 1 above)
Comment (maxhelias) [reply setup/_update_recipes.rst.inc]: Indeed, I agree with this change

**Review by xabbuh — APPROVED** [review]

**Comment by wouterj** [comment]
Thanks for your work on this topic, Maxime!

**Pull request merged**